lib/opensearch.rb

```diff
@@ -31,7 +31,19 @@ def new(url)
 
       private
       def fetch_description(url)
-        content = open(url) {|f| content = f.read }
+        # content = open(url) {|f| content = f.read }
+        # Skip ssl verification
+        uri =  URI.parse(url)
+        http = Net::HTTP.new(uri.host, uri.port)
+        if uri.scheme == "https"  # enable SSL/TLS
+          http.use_ssl = true
```

Comment:
you might have to add this in case of https `http.verify_mode = OpenSSL::SSL::VERIFY_NONE`